packages/react-tinacms-inline/src/inline-form.tsx

```diff
@@ -44,9 +46,20 @@ export interface InlineFormState {
   setFocussedField(field: string): void
 }
 
-export function InlineForm({ form, children }: InlineFormProps) {
+export interface ExtendedInlineFormState extends InlineFormState {
```

Comment:
What's this interface for?

Reply:
I just don't see it being used in here? You may extend it all you want, but as far as I know, you will still be incompatible with the original since you're introducing a new property.